insights/parsers/cifs.py

```diff
@@ -0,0 +1,79 @@
+"""
+Get CIFS Info
+=============
+
+This module contains the following parsers:
+
+CIFSDebugData - file ``/proc/fs/cifs/DebugData``
+================================================
```

Comment:
Better use `---` instead of `===` for such cases, eg: https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/ip.py#L8 .